packages/theme-chalk/src/menu.scss

```diff
@@ -58,7 +58,11 @@
   padding-left: 0;
   background-color: getCssVar('menu-bg-color');
   box-sizing: border-box;
-
+  & .#{$namespace}-menu-item:not(.is-disabled):hover,
+  & .#{$namespace}-menu-item:not(.is-disabled):focus {
```

Comment:
I think it should be possible to cancel focus, e.g [element-ui menu](https://element.eleme.io/#/zh-CN/component/menu)